Source/WebCore/style/StyleResolver.cpp

```diff
@@ -136,6 +139,8 @@ class Resolver::State {
     const RenderStyle* m_rootElementStyle { nullptr };
 
     std::unique_ptr<RenderStyle> m_userAgentAppearanceStyle;
+
+    AnchorPositionedStateMap* m_anchorPositionedStateMap { nullptr };
```

Comment:
You shouldn't need `nullptr` here, just `{ };` is fine.

Reply:
Would it be better to change all of the private members to remove `nullptr` for consistency? For context, the entire list of members is
```c++
private:
    const Element* m_element { nullptr };
    std::unique_ptr<RenderStyle> m_style;
    const RenderStyle* m_parentStyle { nullptr };
    std::unique_ptr<const RenderStyle> m_ownedParentStyle;
    const RenderStyle* m_rootElementStyle { nullptr };

    std::unique_ptr<RenderStyle> m_userAgentAppearanceStyle;

    AnchorPositionedStateMap* m_anchorPositionedStateMap { nullptr };
```

Reply:
Sure, that would be a good change.